debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsSnapshotProducerIT.java

```diff
@@ -551,6 +551,42 @@ public void shouldNotSnapshotMaterializedViews() throws Exception {
         consumer.process(record -> assertReadRecord(record, expectedValueByTopicName));
     }
 
+    @Test
+    @FixFor("DBZ-1755")
+    public void shouldGenerateSnapshotForPositiveMoney() throws Exception {
```

Comment:
Is there a test for the `null` case, too? Asking because it's explicitly mentioned in the issue.

Reply:
Yep, added.